src/components/reps/value-renderer.jsx

```diff
@@ -134,37 +130,35 @@ export class ValueRenderer extends React.Component {
     if (!this.props.render) {
       return <div className="empty-resultset" />;
     }
-
     const value = renderValue(
       this.props.valueToRender,
       this.props.inContainer,
       this.state.useDefault
     );
-
```

Comment:
I think this should be deprecated for now, since it doesn't have a role at the moment and we're about to switch over to a new set of renderers